test/src/test/java/hudson/cli/SetBuildDescriptionCommandTest.java

```diff
@@ -145,4 +148,14 @@
         assertThat(result.stderr(), containsString("ERROR: No such build #2"));
     }
 
+    //TODO: determine if this should be pulled out into JenkinsRule or something
```

Comment:
It doesn't translate things between Batch and Shell, it says right on the wiki that it doesn't do that. It won't translate variable accesses or commands.